api/URL.json

```diff
@@ -982,10 +982,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/URL/toString",
           "support": {
             "chrome": {
-              "version_added": "52"
+              "version_added": "37"
```

Comment:
```suggestion
              "version_added": "19"
```

Reply:
I'm a bit confused at this one, because up at the top of the file I read it as saying that the URL API itself was only added in version 32.

Hmm I see, I guess the point is that there was a `webkitURL` since version 2, and then that grew this feature in version 19? That makes sense, then.

Reply:
The the data at the top indicates that the URL API was introduced initially with the WebKit prefix in version 19 (well, it says 2, but I disproved this) but then it was unprefixed in Chrome 32.  These features were a part of initial API implementation, hence why I’m suggesting 19.

Reply:
Makes sense! Glad you sorted that out about the version it was originally introduced in, too.

(I wonder what caused the data to originally get entered with a too-early version there. I think I understand a too-late version -- the contributor tests on that version, and not earlier ones -- but too-early is more puzzling. Doesn't matter for fixing it, though.)